docs/manual/mustcall-checker.tex

```diff
@@ -0,0 +1,176 @@
+\htmlhr
+\chapterAndLabel{Must Call Checker}{mustcall-checker}
+
+The Must Call Checker conservatively over-approximates
+the set of methods that an object should call before it is de-allocated.
+It is intended to be run as a subchecker of another checker: on its own, it does not
+enforce any rules other than subtyping.
+
+For example, an object whose type is \<java.io.OutputStream> might
+have an obligation to call the \<close()> method, to release an underlying file resource. Or,
+such an \<OutputStream> might not have such an obligation, if the underlying resource is
+a byte array. The Must Call Checker therefore over-approximates the methods that such an
+\<OutputStream> should call by producing the set containing \<close()>.
+
+\sectionAndLabel{Must Call annotations}{must-call-annotations}
+
+The Must Call Checker supports these type qualifiers, whose subtyping relationships
+are shown in Figure~\ref{fig-must-call-hierarchy}:
+
+\begin{description}
+
+\item[\refqualclasswithparams{checker/mustcall/qual}{MustCall}{String[] value}]
+  indicates that the qualifier's arguments are a superset of the methods that the annotated program element
+  must call before it is de-allocated.
+  The type \<@MustCall(\{"close"\}) OutputStream>, for example, can be read as ``an OutputStream that might need
+  to call close before it is de-allocated.''
+  The default type qualifier for an unannotated type is \<@MustCall(\{\})>.
+
+\item[\refqualclass{checker/mustcall/qual}{MustCallUnknown}]
+  is used internally by the type system but should never be written by a
+  programmer.  It theoretically represents a value with an unknowable or infinite set
+  of must-call methods.
+
+\item[\refqualclasswithparams{checker/mustcall/qual}{InheritableMustCall}{String[] value}]
+  is an alias of \<@MustCall> that can only be written on a class declaration. It adds an \<@MustCall>
+  annotation with the same arguments to
+  the class declaration on which it is written, and also to all subclasses, freeing the user of the need
+  to write the \<@MustCall> annotation on every subclass.
+
+\item[\refqualclass{checker/mustcall/qual}{PolyMustCall}]
+  is the polymorphic annotation for the Must Call type system
+  (Section~\ref{method-qualifier-polymorphism}).
+
+\end{description}
+
+\begin{figure}
+\includeimage{mustcall}{5cm}
+\caption{Subtyping relationships between examples of type qualifiers from the Must Call Checker's type
+hierarchy, for an object with only two methods (\<a> and \<b>).
```

Comment:
I rephrased it, though I still think it could be more concise.

I also changed the examples from "a" and "b" to "foo" and "bar", which are more dissimilar than "methodA" and "methodB" (while also conveniently being short enough to not require me to redraw the figure).